CHANGELOG.md

```diff
@@ -1,3 +1,125 @@
+# v1.38.1
+
+_Notice: The v1.38.0 images were promoted incorrectly due to a proces error. Do not use any images from `v1.38.0` and upgrade directly to `v1.38.1`._
+
+## Changes by Kind
+
+### Urgent Upgrade Notes
+*(No, really, you MUST read this before you upgrade)*
+
+#### Breaking Metrics Changes
+
+Node plugin metrics have been renamed to follow Prometheus best practices:
+- Added `aws_ebs_csi_` prefix
+- Added `_total` suffix for counters
+- Changed time units from microseconds to seconds for all counters
+
+The controller plugin metrics now use the prefix `aws_ebs_csi_` instead of `cloudprovider_aws_`. The old metric names can be disabled via the CLI parameter `--deprecated-metrics=false` on the controller. This will default to `true` in a future version of the EBS CSI Driver. The old metric names (`cloudprovider_aws_*`) are deprecated and will be removed in a future version of the EBS CSI Driver.
```

Comment:
Can we change to this to something like:

```
The controller plugin metrics now use the prefix `aws_ebs_csi_` instead of `cloudprovider_aws_`. The `cloudprovider_aws_` metrics will still be emitted. These old metric names can be disabled via the CLI parameter `--deprecated-metrics=false` on the controller.
```

To explicitly clarify that the controller plugin metrics change isn't breaking yet.